### PR TITLE
[protobuf] protobuf v3.13.0

### PR DESCRIPTION
**Pull request by jozefizso**
Update to Protocol Buffers v3.13.0
https://github.com/protocolbuffers/protobuf/releases/tag/v3.13.0

### CODE REVIEW

File: ports/protobuf/CONTROL [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Source: protobuf
-Version: 3.12.3
-Port-Version: 2
-Homepage: https://github.com/google/protobuf
+Version: 3.13.0
+Port-Version: 1
```
Comment (PhoebeHui) [inline ports/protobuf/CONTROL:3]:
```suggestion
```

**Review by PhoebeHui — COMMENTED** [review]
Thanks for the PR!

**Review by PhoebeHui — APPROVED** [review]

**Comment by BillyONeal** [comment]
The arm-uwp failures look legitimate...

**Comment by jozefizso** [comment]
The build `arm_uwp` failed because of:

```
-- Downloading https://github.com/protocolbuffers/protobuf/archive/fde7cf7358ec7cd69e8db9be4f1fa6a5c431386a.tar.gz...
-- Extracting source D:/downloads/protocolbuffers-protobuf-fde7cf7358ec7cd69e8db9be4f1fa6a5c431386a.tar.gz
-- Applying patch fix-uwp.patch
-- Applying patch fix-android-log.patch
-- Applying patch fix-static-build.patch
-- Using source at D:/buildtrees/protobuf/src/a5c431386a-c9deea9e31.clean
CMake Error at ports/protobuf/portfile.cmake:22 (message):
  Cross-targetting protobuf requires the x86-windows protoc to be available.
  Please install protobuf:x86-windows first.
Call Stack (most recent call first):
  scripts/ports.cmake:79 (include)
```

I'm not sure what to do here. When I build the **protoc:x86-windows** first and **protoc:arm_uwp** second, it works.

**Comment by BillyONeal** [comment]
> I'm not sure what to do here. When I build the **protoc:x86-windows** first and **protoc:arm_uwp** second, it works.

CI currently installs "protobuf:x86-windows" first to make this work: https://github.com/microsoft/vcpkg/blob/c3f69d47c8b1dfe3afeeb96e65674c01e1c82191/scripts/azure-pipelines/test-modified-ports.ps1#L98

Perhaps that was split into "protoc" instead and that wasn't updated?

**Pull request merged**